### DISCUSSION TITLE
Need to Introduce Select Dropdown and Multi Select Drop Down

### DISCUSSION THREAD

**Discussion by MaheRepository** [twbs/bootstrap#40641] [Ideas]
Bootstrap should introduce select drop down convert is search functionality and multiselect dropdown. Like

https://developer.snapappointments.com/bootstrap-select/

https://select2.org/

Now a days its essential user-friendly UI search dropdown huge demand from client. but external libraries are not compatible with bootstrap 5 and latest version.

Bootstrap team should work on it and introduce in new upcoming version.

**Comment by julien-deramond** [comment]
Hey @MaheRepository 
If I'm not wrong, this is referenced as a feature request in the following GitHub issue:  https://github.com/twbs/bootstrap/issues/38271

**Reply by jdanign** [reply]
I think it is a very necessary function, it would be used a lot, and I think that bootstrap should strengthen this component so that the search for options is allowed at least. In the thread you have marked, this request is not even answered.
